nuxeo-features/nuxeo-admin-center/nuxeo-admin-center-jsf/src/test/java/org/nuxeo/ecm/admin/setup/TestSetupWizardActionBean.java

```diff
@@ -100,7 +104,24 @@ public void setUp() throws Exception {
         // WARN [ConfigurationGenerator] Parameter mail.transport.username is deprecated ...
         // WARN [ConfigurationGenerator] Missing value for nuxeo.db.type, using default
         capturedLog.assertHasEvent();
-        assertEquals(6, capturedLog.getCaughtEvents().size());
+
+        String[] expectedFixedEvents = {
+            "Unknown server.",
+            "Server will be considered as not configurable.",
+            "Parameter mail.transport.username is deprecated - please use mail.transport.user instead",
+            "Parameter mail.transport.username is deprecated - please use mail.transport.user instead",
+            "Missing value for nuxeo.db.type, using default"
+        };
+        List<String> fixedIntersect = Arrays.asList(expectedFixedEvents).stream()
```

Comment:
You can use directly:
```suggestion
        List<String> fixedIntersect = Stream.of(expectedFixedEvents)
```